README.md

```diff
@@ -3,19 +3,49 @@
 
 A High Performance [NATS](https://nats.io) Server written in [Go.](http://golang.org)
 
-**Note**: The `master` branch may be in an *unstable or even in a broken state* during development. Please use [releases][github-release] instead of the `master` branch in order to get stable binaries.
+## Quickstart
```

Comment:
Feels like Quickstart would be to download a binary. This needs Go installed which is not quick unless you already have it. Might consider combining this with binary download. We could say "If you have Go installed and want to run directly from tip, ..."